teacher.rb

```diff
@@ -1,6 +1,6 @@
 require './person'
 
-class Teacher < person
+class Teacher < Person
```

Comment:
- [ ] I am getting an error when creating a teacher. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/180239330-d1676648-9b72-480c-876c-c7518193aad0.png)